packages/@aws-cdk/aws-lambda-python-alpha/test/integ.function.poetry.ts

```diff
@@ -17,89 +17,116 @@ class TestStack extends Stack {
 
     const pythonFunction39 = new lambda.PythonFunction(this, 'my_handler_inline', {
```

Comment:
I think it's better to keep the existing test cases as they are and:
1. Add `pythonFunction37WithoutUrls`, `pythonFunction38WithoutUrls`, `pythonFunction39WithoutUrls`
2. Add `pythonFunction310`, `pythonFunction310WithHashes`, `pythonFunction310Excludes`, `pythonFunction310WithoutUrls`, `pythonFunction311`, `pythonFunction311WithHashes`, `pythonFunction311Excludes`, `pythonFunction311WithoutUrls` for 3.10 and 3.11

Reply:
@lpizzinidev The older versions of Python are erroring out:

```
{
  "ExecutedVersion": "$LATEST",
  "FunctionError": "Unhandled",
!! Expected 200 but received {"errorMessage": "Unable to import module 'index': urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'OpenSSL 1.0.2k-fips  26 Jan 2017'. See: https://github.com/urllib3/urllib3/issues/2168", "errorType": "Runtime.ImportModuleError", "stackTrace": []}
  "Payload": "{\"errorMessage\": \"Unable to import module 'index': urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'OpenSSL 1.0.2k-fips  26 Jan 2017'. See: https://github.com/urllib3/urllib3/issues/2168\", \"errorType\": \"Runtime.ImportModuleError\", \"stackTrace\": []}",
  "StatusCode": 200
}
```

I can look at updating dependencies if you want?

Reply:
I'm not a Python expert, but maybe this could be solved by creating two handlers.
A `lambda-handler-poetry` with `urllib3` v1.x for runtimes with Python <= 3.9 and a `lambda-handler-poetry` with `urllib3` v2.x for runtimes with Python >= 3.10.
Not sure if it'll work or if it's the right approach.

Reply:
Done. Added a simple Python function with no dependencies.